Symfony/CS/Tokenizer/Tokens.php

```diff
@@ -135,9 +135,11 @@ public static function fromCode($code)
 
         $tokens = token_get_all($code);
 
-        foreach ($tokens as $index => $tokenPrototype) {
-            $tokens[$index] = new Token($tokenPrototype);
+        foreach ($tokens as $index => &$tokenPrototype) {
+            $tokenPrototype = new Token($tokenPrototype);
```

Comment:
@stof : I'm really concerned now, I shall revert all that references.

//cc @keradus